### ISSUE TITLE
Add a warning to readme about using `model` in Octane

### ISSUE THREAD

**Issue by ijlee2** [ronco/ember-cli-head#70]
Hello. A couple of weeks ago, my team introduced a bug when we Octanified our templates by changing all instances of `this.model` to `@model`. We didn't realize until today that we should have left alone `this.model` in `app/templates/head.hbs`, since `model` refers to the `head-data` service.

I was wondering if you could make this clear in the readme file, so that other developers and teams may avoid our mistake when using Octane?

**Comment by rwwagner90** [comment]
I think we could even go a step further and not name it `model` to avoid confusion, but at the very least, I think we should do as @ijlee2 suggests.

**Comment by ijlee2** [comment]
Sounds good. I can make a PR this weekend to add the warning and update the example code.

I think renaming `model` to a more descriptive name would be good. However, since this will cause a breaking change, it may be out of the extent of help that I can offer.

**Comment by rwwagner90** [comment]
@ijlee2 yeah, I think no need to introduce a breaking change right now, but we do need to guard people against making this mistake.

**Comment by rwjblue** [comment]
Ya, we can expose a new field though. So you could author `this.headData.someProperty`, instead of `this.model`. Using `model` here really just isn't very useful.

**Comment by ijlee2** [comment]
I opened the PR at https://github.com/ronco/ember-cli-head/pull/81.
